src/screens/CreatePost/elements/SheetAddTopic.js

```diff
@@ -143,9 +143,9 @@ const SheetAddTopic = ({refTopic, onAdd, topics, onClose, chatTopics}) => {
                     if (v.match(/\s+$/gm)) {
                       return add();
                     }
-                    setTopic(v);
+                    setTopic(v?.toLowerCase());
                     if (v !== '') {
-                      searchTopic(v);
+                      searchTopic(v?.toLowerCase());
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for enforcing lowercase letters in the topic. However, there's a potential issue with null or undefined values. If `v` is null or undefined, calling `toLowerCase()` on it will throw an error. To prevent this, you should add a check to ensure `v` is not null or undefined before calling `toLowerCase()`. Here's how you can do it:

```diff
- setTopic(v?.toLowerCase());
+ setTopic(v ? v.toLowerCase() : v);
```

And

```diff
- searchTopic(v?.toLowerCase());
+ searchTopic(v ? v.toLowerCase() : v);
```
This way, if `v` is null or undefined, it will remain as it is and won't cause any errors.


<!-- This is an auto-generated comment by OSS CodeRabbit -->